src/components/Toast/Toast.js

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+import { useEffect } from "react";
+// import { useStateContext } from "../../context/StateProvider";
+import "./toast.css";
+
+export const Toast = () => {
+  // const { toast, toastHandler } = useStateContext();
+  // const { showToast, message, type } = toast;
+
+  // useEffect(() => {
+  //   const timeout = setTimeout(() => {
+  //     toastHandler();
+  //   }, 3000);
+  //   return () => clearTimeout(timeout);
+  // }, [showToast]);
```

Comment:
why so much code is commented? will it required later? if not then remove it

Reply:
The commented-out code will be used later. These lines were previously used in my projects so I just copy-pasted them here.